package/piksi_ins/overlay/etc/init.d/S96piksi_ins

```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
 
 name="PoseDaemon"
-cmd="PoseDaemon"
+cmd="start_pose_daemon"
 dir="/"
+# User is set to posed in the start_pose_daemon wrapper since we
+#   need to process things with nap_wrap before launching.
 user=""
```

Comment:
I had already setup a "piksi_ins" user for PoseDaemon I think.  Did we get rid of that?

Reply:
See here: https://github.com/swift-nav/piksi_buildroot/pull/910/files#diff-552ec8890331de7822cdf5ad3c619d09L7

Will the posed user exist?

Reply:
Maybe we need to remove this user from .mk

Reply:
We can leave it in the .mk and change posed->piksi_ins (and get rid of the dynamic user creation), I had it as a dynamic user since the devmem group was initially dynamic, but it isn't any more.

Reply:
Fixed